lib/urbanopt/reporting/default_reports/schema/scenario_schema.json

```diff
@@ -130,11 +130,12 @@
 					"type": "string"
 				},
 				"location": {
-					"description": "Location of PV. Available options are roof and ground.",
+					"description": "Location of PV. Available options are roof, ground or both.",
 					"type": "string",
 					"enum": [
 						"roof",
-						"ground"
+						"ground",
+						"both"
```

Comment:
sorry...can you change "both" to:  "roof and ground". ?  that way it's more descriptive in the geojson file. thanks! other than that looks gerat!

Reply:
I took this from the reopt input schema https://developer.nrel.gov/docs/energy-optimization/reopt-v1/#PV_collapsecontainer

Reply:
ok, let's keep it if it matches their schema then. thanks!